docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -143,6 +143,114 @@ Elasticsearch Trino
 (all others)  (unsupported)
 ============= =============
 
+.. _elasticesearch-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and Elasticsearch each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+Elasticsearch type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Elasticsearch types to the corresponding Trino types
+according to the following table:
+
+.. list-table:: Elasticsearch type to Trino type mapping
+  :widths: 30, 30, 50
+  :header-rows: 1
+
+  * - Elasticsearch type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``DOUBLE``
+    - ``DOUBLE``
+    -
+  * - ``FLOAT``
+    - ``REAL``
+    -
+  * - ``BYTE``
+    - ``TINYINT``
+    -
+  * - ``SHORT``
+    - ``SMALLINT``
+    -
+  * - ``INTEGER``
+    - ``INTEGER``
+    -
+  * - ``LONG``
+    - ``BIGINT``
+    -
+  * - ``KEYWORD``
+    - ``VARCHAR``
+    -
+  * - ``TEXT``
+    - ``VARCHAR``
+    -
+  * - ``DATE``
+    - ``TIMESTAMP``
+    - For more information, see :ref:`elasticsearch-date-types`.
+  * - ``IPADDRESS``
+    - ``IP``
+    -
+
+No other types are supported.
+
+Trino type to Elasticsearch type mapping
```

Comment:
This section duplicates the one above. The mapping is a correspondence of types between Trino and Elasticsearch, so it doesn't make sense to have two tables for Trino->ES and ES->Trino

Reply:
Since this a read-only connector, removed the second table.